docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+## Step one: Decide on your network configuration
+
+The structure of a blockchain network must be dictated by the use case. These fundamental business decisions will vary according to your use case, but let's consider a few scenarios.
+
+In contrast to development environments or proofs of concept, security, resource management, and high availability become a priority when operating in production. How many nodes do you need to satisfy high availability, and in what data centers do you wish to deploy them in to satisfy both the needs of disaster recovery and data residency? How will you ensure that your private keys and roots of trust remain secure?
+
+In addition to the above, here is a sampling of the decisions you will need to make before deploying components:
+
+* **Certificate Authority configuration**.
+
+  As part of the overall decisions you have to make about your peers (how many, how many on each channel, and so on) and about your ordering service (how many nodes, who will own them), you also have to decide on how the CAs for your organization will be deployed. Do you want to use TLS? Then you will need a TLS CA, and it will need to be deployed before your enrollment CA. We'll discuss this more in [Step 3: Set up your CAs](#step-3-set-up-your-cas).
+
+* **Use Organizational Units or not?**
+
+  Some organizations might find it necessary to establish Organizational Units to create a separation between certain identities and MSPs created by a single CA.
+
+* **Database type.**
+
+  Some channels in a network might require all data to be modeled in a way [CouchDB](couchdb_as_state_database.html) can understand, while other networks, prioritizing speed, might decide that all peers will use LevelDB.
+
+* **Channels and private data.**
+
+  Some networks might decide that [channels](channels.html) are the best way to ensure privacy and isolation for certain transactions. Others might decide that a single channel, along with [private data collections](private-data/private-data.html), better serves their need for privacy.
+
+* **Container orchestration.**
+
+  Different users might also make different decisions about their container orchestration, creating separate containers for their peer process, logging for the peer, CouchDB, gRPC communications, and chaincode, while other users might decide to combine some of these processes.
+
+* **External chaincode.**
+
+  The new [external chaincode service](cc_service.html) will also be implemented by some users and not others.
```

Comment:
Isn't there a three way choice here? The user needs to choose between:
- Built in build and run support
- Customised build and run using external builders
- Chaincode as a service

From the existing text, it's not clear what the alternative to the external chaincode service is, whereas the other points say "x or y". It would be better to give all three, or make a more general point about users having to make a decision about how they build and run chaincode.

Reply:
I was just providing an example and not a definitive list here but fair enough. Might be better to give the extra links here and a fuller description.